app/rql.py

```diff
@@ -27,22 +27,22 @@ class AuditableMixin(TimestampMixin):
     deleted_by = RelationshipRule(alias="events.deleted.by", rules=ActorRules())
 
 
-class EntitlementRules(ModelRQLRules, AuditableMixin):
-    __model__ = Entitlement
+class AccountRules(ModelRQLRules, AuditableMixin):
+    __model__ = Account
 
     id = FieldRule()
     name = FieldRule()
-    datasource_id = FieldRule()
+    external_id = FieldRule()
+    type = FieldRule()
     status = FieldRule()
 
 
-class AccountRules(ModelRQLRules, AuditableMixin):
-    __model__ = Account
+class SystemRules(ModelRQLRules, AuditableMixin):
+    __model__ = System
 
     id = FieldRule()
-    name = FieldRule()
     external_id = FieldRule()
-    type = FieldRule()
+    owner_id = RelationshipRule(rules=AccountRules(), alias="owner_id")
```

Comment:
this should be just owner and no need to add an alias. If you want to filter by owner id you should use for example `eq(owner.id,FACC-1234-5678)`